apps/bandwagon/templates/bandwagon/includes/addedit.html

```diff
@@ -1,6 +1,11 @@
 {{ csrf() }}
 
 <fieldset>
+  {# Honeypot field for bots to get caught #}
+  {% if form.your_name %}
```

Comment:
And (imho) helps seeing that this field might not always exist (e.g it doesn't for edit)